kedge/auth.yml

```diff
@@ -0,0 +1,18 @@
+name: auth
+containers:
+- image: docker.io/fabric8/fabric8-auth:latest
+  env:
+  - name: AUTH_WIT_URL
+    value: "http://192.168.42.1:8080"
```

Comment:
Could we use the WIT service name here instead of the url ? That way we'll avoid using IP addresses, and make we can use these templates to run on any OpenShift ( dev/integration ) cluster as well.

Reply:
till now, WIT is not running as  a service, from makefile we are only running auth, db-auth and db.
And core is running locally. so can't give service name there, need to see other better way instead of IP. maybe `minishift ip` command if it works inside kedge file :smiley:

Reply:
you can pass env variables into Kedge.

you can put this in your kedge file:
```
value: "[[ AUTH_WIT_URL ]]"
```

and than call Kedge like this:
```
AUTH_WIT_URL=http://192.168.42.1:8080 kedge apply -f file.yml
```

Reply:
@pranavgore09 Ah OK, seeing this confused me a bit https://github.com/pranavgore09/fabric8-wit/blob/9f9a3e2e122bf0f28ced5ca2cb7d2e4ec3182645/kedge/core.yml

 you already have most of it in place I think :)

Reply:
thanks @kadel done in here https://github.com/fabric8-services/fabric8-wit/pull/1741/commits/ea3bbdcb14d6409ccf9b53d78364bd90e621ae21